File: sibyl-discord.js
import { REST, Routes, Collection, Client, GatewayIntentBits, Events } from "discord.js";

import sibylCommand from "./commands/sibyl.js";
import dominatorCommand from "./commands/dominator.js";
import psychopassCommand from "./commands/psychopass.js";

import messageCreate from "./events/messageCreate.js";
import interactionCreate from "./events/interactionCreate.js";
import ready from "./events/ready.js";
import guildDelete from "./events/guildDelete.js";
import guildCreate from "./events/guildCreate.js";
import guildMemberAdd from "./events/guildMemberAdd.js";

const client = new Client({
    intents: [
        GatewayIntentBits.Guilds,
        GatewayIntentBits.GuildMessages,
        GatewayIntentBits.GuildBans,
        GatewayIntentBits.MessageContent
    ]
});
const rest = new REST({ version: "10" }).setToken(process.env.DISCORD_BOT_TOKEN);
const commands = [sibylCommand, dominatorCommand, psychopassCommand];

await rest.put(Routes.applicationCommands(process.env.DISCORD_CLIENT_ID), { body: commands.map(command => command.data) });
console.log('Successfully Reloaded Application (/) Commands.');
client.commands = new Collection();
commands.map(command => client.commands.set(command.data.name, command));
console.log("Successfully Registered Application (/) Command Actions.")

client.on(Events.ClientReady, ready);
client.on(Events.GuildCreate, guildCreate);
client.on(Events.GuildDelete, guildDelete);
client.on(Events.GuildMemberAdd, guildMemberAdd);
client.on(Events.MessageCreate, messageCreate);
client.on(Events.InteractionCreate, interactionCreate);

client.login(process.env.DISCORD_BOT_TOKEN);

File: sibyl-discord.py
from os import getenv
from loguru import logger as log
from discord import Bot, Intents

from commands.sibyl import Sibyl
from commands.psycho_pass import PsychoPass
from commands.dominator import Dominator
from events.ready import Ready
from events.guild import GuildEvents
from events.member import MemberEvents
from events.message import MessageEvents


intents = Intents.default()
intents.guilds = True
intents.guild_messages = True
intents.moderation = True
intents.members = True
intents.message_content = True
bot = Bot(intents=intents)


cogs = [Sibyl, PsychoPass, Dominator, Ready, GuildEvents, MemberEvents, MessageEvents]
for cog in cogs:
    bot.add_cog(cog(bot))
    log.debug("registered cog: {}", cog.__name__)

log.info("starting sibylmod discord bot...")
bot.run(getenv("DISCORD_BOT_TOKEN"))